<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>hydrabus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/index.png</normaloff>:/res/index.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="ControlWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>7</number>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="PWM" name="tabPWM">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <attribute name="title">
            <string>PWM</string>
           </attribute>
          </widget>
          <widget class="ADC" name="tabADC">
           <attribute name="title">
            <string>ADC</string>
           </attribute>
          </widget>
          <widget class="DAC" name="tabDAC">
           <attribute name="title">
            <string>DAC</string>
           </attribute>
          </widget>
          <widget class="GPIO" name="tabGPIO">
           <attribute name="title">
            <string>GPIO</string>
           </attribute>
          </widget>
          <widget class="SPI" name="tabSPI">
           <attribute name="title">
            <string>SPI</string>
           </attribute>
          </widget>
          <widget class="I2C" name="tabI2C">
           <attribute name="title">
            <string>I2C</string>
           </attribute>
          </widget>
          <widget class="UART" name="tabUART">
           <attribute name="title">
            <string>UART</string>
           </attribute>
          </widget>
          <widget class="JTAG" name="tabJTAG">
           <attribute name="title">
            <string>JTAG</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Console" name="ConsoleWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QWidget</extends>
   <header location="global">console.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PWM</class>
   <extends>QWidget</extends>
   <header location="global">pwm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ADC</class>
   <extends>QWidget</extends>
   <header location="global">adc.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DAC</class>
   <extends>QWidget</extends>
   <header location="global">dac.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GPIO</class>
   <extends>QWidget</extends>
   <header location="global">gpio.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPI</class>
   <extends>QWidget</extends>
   <header location="global">spi.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>I2C</class>
   <extends>QWidget</extends>
   <header location="global">i2c.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UART</class>
   <extends>QWidget</extends>
   <header location="global">uart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JTAG</class>
   <extends>QWidget</extends>
   <header location="global">jtag.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setTabIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setTabIndex(int)</slot>
 </slots>
</ui>
